projects/demo/src/modules/components/rating/rating.component.ts

```diff
@@ -0,0 +1,43 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {RawLoaderContent, TuiDocExample} from '@taiga-ui/addon-doc';
+
+@Component({
+    selector: 'example-rating',
+    templateUrl: './rating.template.html',
+    changeDetection,
+})
+export class ExampleTuiRatingComponent {
+    rateColor: string = 'var(--tui-accent)';
```

Comment:
Why not just `color`? This is how we always did it. And we use `Variants` postfix for options.